tests/drivers/build_all/sensor/uart.dtsi

```diff
@@ -15,3 +16,13 @@ test_uart_mhz19b {
 test_uart_pms7003 {
 	compatible = "plantower,pms7003";
 };
+
+test_uart_grow_r502a {
+	#address-cells=<1>;
+	#size-cells=<0>;
+	grow_r502a@0 {
```

Comment:
Why does this have an address?

Reply:
basically, this fingerprint sensor creates an RX packet based on the address we are going to pass, initially, it was passed to the driver via the address parameter from the fps node and then based on the PR suggestion from @gmarull  [here](https://github.com/zephyrproject-rtos/zephyr/pull/40394#discussion_r943217396)  address is given to driver via reg property. if you are having any conflict of opinion please let me know. thanks